client/state/selectors/get-site-design-type.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSite } from 'state/sites/selectors';
+
+/**
+ * @param {Object} state Global state tree
+ * @param {Number} siteIdOrSlug Site ID
+ * @return {String} The design type of the site
+ */
+export default function getSiteDesignType( state, siteIdOrSlug ) {
+	if ( ! siteIdOrSlug ) {
+		return null;
+	}
+
+	const site = getSite( state, siteIdOrSlug );
+	if ( ! site ) {
+		return null;
+	}
+
+	return get( site, 'options.design_type', '' );
```

Comment:
You can leverage `getSiteOption` like this:

https://github.com/Automattic/wp-calypso/blob/8eb8c95465f063b5b56e061d6b8eb5ffc1077d3c/client/my-sites/stats/checklist-banner/index.jsx#L201

Reply:
I will remove the file in favor of `getSiteOption`. Thanks. :)